Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2789,6 +2789,21 @@ FullScreenEnabled:
     WebCore:
       default: false
 
+FullScreenKeyboardLock:
+  type: bool
+  status: developer
```

Comment:
Should be testable, developer is for actual developer tools settings.

Reply:
@nt1m Thanks for the feedback. Updated.